src/components/RangeSlider/components/DualThumb/tests/DualThumb.test.tsx

```diff
@@ -20,15 +19,15 @@ describe('<DualThumb />', () => {
   describe('id', () => {
     it('is used to set the id on the wrapper div', () => {
       const dualThumb = mountWithAppProvider(
-        <DualThumb {...mockProps} id="RangeSlider" />,
+        <DualThumb {...mockProps} value={[0, 1]} />,
```

Comment:
If this is testing the `id`, I don't understand why `id="RangeSlider"` was removed here?

Reply:
It is already accounted for in `mockProps`, so was unnecessary to override here. Also whatever we are passing in as `mockProps` would wipe out conflicting values passed as props. Unless we are passing something different from what we passed to mockProps, it's fine just to use whatever we used there.